### PR TITLE
Add tokens to lock keeper function

### PR DESCRIPTION
**Pull request by antstalepresh**

### CODE REVIEW

File: x/lockup/keeper/lock.go [hunk 1]
```diff
@@ -281,6 +281,40 @@ func (k Keeper) UnlockPeriodLockByID(ctx sdk.Context, LockID uint64) (*types.Per
 	return lock, err
 }
 
+// AddTokensToLock locks more tokens into a lockup
+func (k Keeper) AddTokensToLock(ctx sdk.Context, owner sdk.AccAddress, lockID uint64, coins sdk.Coins) (*types.PeriodLock, error) {
```
Comment (ValarDragon) [inline x/lockup/keeper/lock.go:285]: Can we separate this out into two functions:

```
AddTokensToLockID(ctx sdk.Context, owner sdk.AccAddress, lockID uint64, coins sdk.Coins)
``` 
and 
```
AddTokensToLock(ctx, lock *types.PeriodLock, coins sdk.Coins)
```

I think both will come in handy

File: x/lockup/keeper/lock.go [hunk 2]
```diff
@@ -281,6 +281,40 @@ func (k Keeper) UnlockPeriodLockByID(ctx sdk.Context, LockID uint64) (*types.Per
 	return lock, err
 }
 
+// AddTokensToLock locks more tokens into a lockup
```
Comment (ValarDragon) [inline x/lockup/keeper/lock.go:284]:
```suggestion
// AddTokensToLock locks more tokens into a lockup
// This also saves the lock to the store.
```

**Review by ValarDragon — APPROVED** [review]
LGTM, just some minor comments

File: x/lockup/keeper/lock.go (hunk 1 above)
Comment (antstalepresh) [reply x/lockup/keeper/lock.go]: Did split the function into two pieces

**Pull request merged**